app/views/laundry_rooms/index.html.erb

```diff
@@ -55,3 +55,7 @@
       <% end %>
     </table>
 <% end %>
+
+<%= month_calendar do |date| %>
+    <%= date.day %>
+<% end %>
```

Comment:
Please do not change anything in files that are not related to what you are doing. In this case you will end up with merge conflicts later. Focus on getting the navbar working

Reply:
yeah I'll take it out